src/adapter/src/session/vars.rs

```diff
@@ -379,6 +379,18 @@ const PERSIST_COMPACTION_MINIMUM_TIMEOUT: ServerVar<Duration> = ServerVar {
     safe: true,
 };
 
+/// Controls whether or not to use the new storage `persist_sink` implementation in storage
+/// ingestions.
+const ENABLE_MULTI_WORKER_STORAGE_PERSIST_SINK: ServerVar<bool> = ServerVar {
```

Comment:
Nit: isn't `ENABLE_MULTI_WORKER_PERSIST_SINK` or `ENABLE_MULTI_WORKER_STORAGE_SINK` equally descriptive? Do we differentiate between storage persist sinks and dataflow persist sinks?

Reply:
there are:
- storage sinks (CREATE SINK)
- storage persist sinks (used for sources)
- compute persist sinks (used for MV's)

and this only applies to the first one!